pkg/kubelet/dockertools/manager.go

```diff
@@ -79,13 +79,21 @@ const (
 	// networking). Must match the value returned by docker inspect -f
 	// '{{.HostConfig.NetworkMode}}'.
 	namespaceModeHost = "host"
+
+	dockerV110APIVersion = "1.22"
```

Comment:
nit: add a comment explaining this is the api version of docker v1.10 (just because I had some trouble parsing V110 when I first saw it).